server/paymentProviders/transferwise/index.ts

```diff
@@ -73,56 +79,60 @@ async function payExpense(connectedAccount, payoutMethod, expense): Promise<any>
   return { quote, recipient, transfer, fund };
 }
 
-async function getRequiredBankInformation(host: any, currency: string): Promise<any> {
-  const cacheKey = `transferwise_required_bank_info_${host.id}_to_${currency}`;
+async function getAvailableCurrencies(host: any): Promise<any> {
+  const cacheKey = `transferwise_available_currencies_${host.id}`;
   const fromCache = await cache.get(cacheKey);
   if (fromCache) {
     return fromCache;
   }
 
-  const connectedAccount = await models.ConnectedAccount.findOne({
-    where: { service: 'transferwise', CollectiveId: host.id, deletedAt: null },
-  });
+  const connectedAccount =
+    host.connectedAccount?.service === 'transferwise'
```

Comment:
What is this syntax?

Reply:
This is the optional chaining syntax.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
This is the optional chaining syntax.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
@znarf apparently our Babel version already support it too, so we can also use it in API & Frontend JS files